src/content/bridges/index.md

```diff
@@ -113,14 +113,16 @@ User's funds are at risk if:
 - the bridge operators have malicious intent in a trusted bridge
 - the bridge gets hacked
 
-One recent hack was Solana’s Wormhole bridge, [where 120k wETH ($325 million USD) was stolen during the hack](https://rekt.news/wormhole-rekt/).
+One recent hack was Solana’s Wormhole bridge, [where 120k wETH ($325 million USD) was stolen during the hack](https://rekt.news/wormhole-rekt/). Many of the [top hacks in blockchains involved bridges](https://rekt.news/leaderboard/).
 
-Bridges are crucial to onboarding users onto Ethereum L2s, and even for users who want to explore different ecosystems. However, given the risks involved in interacting with bridges, users must understand the trade-offs the bridges are making.
+Bridges are crucial to onboarding users onto Ethereum L2s, and even for users who want to explore different ecosystems. However, given the risks involved in interacting with bridges, users must understand the trade-offs the bridges are making. These are some [strategies for cross-chain security](https://blog.debridge.finance/10-strategies-for-cross-chain-security-8ed5f5879946).
```

Comment:
I'm a little sceptical of this source (incentive and bias), but more so its placement here. In my mind, it sort of implies that if a bridge is built properly then it is invulnerable to cross-chain attacks. Maybe this would be better as a further reading article?